l10n_it_pec_messages/model/mail_thread.py

```diff
@@ -85,11 +85,8 @@ def parse_daticert(self, cr, uid, daticert, context=None):
                         msg_dict['recipient_addr'] = child2.text
         return msg_dict
 
-    def get_pec_attachments(self, cr, uid, message, context=None):
-        postacert = False
-        daticert = False
-        smime = False
-        for part in message.walk():
+    def get_pec_attachments(self, msg, parts={}, num=0):
+        for n, part in enumerate(msg.get_payload()):
```

Comment:
Mi pare che quando lo chiami la prima volta da ```message_parse```, lo chiami in ogni caso, indipendetemente da multipart